domains/rajveer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kyouhei14",
+        "email": "rajveerkobot@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://rajveer-is-dead.netlify.app"
```

Comment:
you need to remove https:// in the CNAME record (also we dont support Netlify at the moment)